web/docs/sizing.md

```diff
@@ -0,0 +1,57 @@
+##### Table of Contents
+
+[Introduction](#Introduction)  
+[8pt grid system](#8pt-grid-system)  
+[Pixels and Accessibility](#pixels-and-accessibility)
+
+## Introduction
+
+Documentation for sharing UX/UI design decisions for the ameliorate project.
+
+# 8pt grid system
+
+The 8pt Grid principle states that multiples of 8 (8, 16, 24, 32, 40, 48, 56, etc.) are used for the layout, dimension, pad, and margin of elements.
```

Comment:
A lot of this is taken directly from the first article. I think we should mention the article first, and mention that we're just summarizing. Also, I think we can summarize the `How to use it` even further
```suggestion
These two blog posts offer reasons for sticking to sizes in multiples of 8 - check them out, but the points that seem most relevant are summarized in the below sections.

### How to use it

- Generally try to use px's that are multiples of 8 (e.g. 8, 16, 24, ...). For the case of rems, these will be multiples of 0.5.
- Font size is an exception, use what looks nice.
- For narrow spacings (padding, margin, line height), multiples of 4 can also work.
- Treat this as a default, but if something looks better without meeting the guideline, just go with that.

### Why?
[I think this section is good as is]
```